README.md

```diff
@@ -469,35 +465,43 @@ Fully customisable.  |
 :star2: | Name | Description | 🌍
 --- | --- | --- | ---
 2 | [@mkatircioglu](https://github.com/mkatircioglu)/[**dotfiles**](https://github.com/mkatircioglu/dotfiles) |  |
-### Crystal #
```

Comment:
@lacivert That's neat too, but I use [`blah`](https://github.com/IonicaBizau/blah). You can read more in this blog post: https://ionicabizau.net/blog/33-how-i-npm

What I did last week was an automatic update across the repositories which updated the README files and some other generated files (e.g. the year in the `LICENSE` files). :book: :rocket: